src/components/com_tjcertificate/site/views/certificate/tmpl/default.php

```diff
@@ -14,6 +14,10 @@
 use Joomla\CMS\Factory;
 use Joomla\CMS\Router\Route;
 use Joomla\CMS\Language\Text;
+use Joomla\CMS\Uri\Uri;
+use Joomla\CMS\HTML\HTMLHelper;
+
+HTMLHelper::_('stylesheet', 'components/com_tjcertificate/assets/font-awesome-4.1.0/css/font-awesome.min.css');
```

Comment:
@praneettekdi @tushar33 do we really need to load entire fa.css just for 3 icons?

Reply:
Yes, I have had a discussion with the designer

Reply:
if you need to use it, instead of adding css in this component, load it from strapper.
Since strapper is installed with most of our other extensions, and it already has FA css

Reply:
I think strapper we are going to remove it from most of our extension. I remember such discussion with Ashwin. @coolbung you please confirm?

Reply:
we will only remove php part of it not common css, js part of it

Reply:
Still, I think we should use the CSS here only as this is a public repo, and putting dependencies from other repo is not a good idea. Let me know your thoughts.

Reply:
@manojLondhe Can you please update what needs to follow?